Kiosk/Bid Fulfillment/KeypadViewModel.swift

```diff
@@ -1,84 +1,78 @@
 import Foundation
+import Action
 import RxSwift
 
 let KeypadViewModelMaxIntegerValue = 10_000_000
 
 class KeypadViewModel: NSObject {
     
-    dynamic private var intValue: Int = 0
-    dynamic private var stringValue: String = ""
+    //MARK: - Variables
     
-    //MARK: - Signals
+    lazy var intValue = Variable(0)
     
-    lazy var intValueSignal: RACSignal = {
-        RACObserve(self, "intValue")
-    }()
-    
-    lazy var stringValueSignal: RACSignal = {
-        RACObserve(self, "stringValue")
-    }()
+    lazy var stringValue = Variable("")
     
-    // MARK: - Commands
+    // MARK: - Actions
     
-    // I have no idea why, but if you try and use `[weak self]` in the closure definition of a RACCommand, the compiler segfaults ¯\_(ツ)_/¯
+    // I have no idea why, but if you try and use `[weak self]` in the closure definition of a Action, the compiler segfaults ¯\_(ツ)_/¯
```

Comment:
It was a compiler thing, but that was with Swift 1.0. Definitely worth trying again.